pyirf/spectral.py

```diff
@@ -195,25 +195,32 @@ class LogParabola:
         :math:`\beta`
     e_ref: astropy.units.Quantity[energy]
         :math:`E_\text{ref}`
+    from_log10: bool
+        If True, compute the energy ration in the exponent factor
+        using base 10, else use natural logarithm.
```

Comment:
Maybe here say that default it base 10 log

Reply:
I don't particularly like the keyword name here, why not call this "log_base" or just pass the log function?

E.g. `log=np.log10` vs. `log=np.log`.

Reply:
Ah, I think you took the name from https://docs.gammapy.org/0.19/api/gammapy.modeling.models.LogParabolaSpectralModel.html#gammapy.modeling.models.LogParabolaSpectralModel.from_log10


But this is not a keyword, it's an alternative constructor, that's why it starts with `from_`.

Reply:
yes, the idea was to keep a nomenclature analog to `gammapy` which is what people tends to use more lately

Reply:
But it doesn't make sense to use the name of a class method for the name of a keyword

Reply:
You could implement the same (or rather the inverse) logic as gammapy, e.g. offer a `from_ln` method that converts the parameters to base 10. 

Or you change the name to `log`, maybe with an enum like this:

```
class LogType(Enum):
    LOG10 = auto()
    LN = auto()

class LogParabola:
    def __init__(self, ..., log=LogParabolaBase.LOG10)

```